fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4980,13 +4983,18 @@ class FMCMainDisplay extends BaseAirliners {
         return 0 <= fuel && fuel <= 80;
     }
 
-    /**
-     *
-     * @returns {*}
+    /**NOTE: This is essentially the 'top level/master' injection point for the current on board fuel.
+     *ALL fuel projections, current fuel level displays etc use the figure returned here as the baseline.
+     * @returns {either the detected fuel in the tanks(if at least one engine running) or pilot entered block fuel(engines off)}
      */
```

Comment:
Describe the units, and the return types, with proper JSDoc format.
```suggestion
    /**
     * Gets the current fuel on board in tons.
     * The FQI system FoB is used when an engine is running, otherwise the block fuel entered by the pilot in the FMS.
     * @returns {number | undefined} fuel on board in tons, or undefined when not available.
     */
```

Reply:
Ok will do.  I put the detail as I felt the initial injection point(and the class) was extremely difficult to find/trace.